doc/whats_new/v0.22.rst

```diff
@@ -308,6 +308,10 @@ Changelog
   removes accents from strings that are in NFKD normalized form. :pr:`15100` by
   :user:`Daniel Grady <DGrady>`.
 
+- |Fix| Fixed a bug that caused :class:`feature_extraction.DictVectorizer` to raise
+  an `OverflowError` during the `transform` operation when producing a `scipy.sparse`
+  matrix on large input data. :pr:`` by :user:`Norvan Sahiner <norvan>`.
```

Comment:
```suggestion
  matrix on large input data. :pr:`15463` by :user:`Norvan Sahiner <norvan>`.
```

Reply:
Whoops, forgot to push it! It should be in there now!

Thank you so much for the reviews and the help on this!